recipes/onnxruntime/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "onnxruntime" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/microsoft/{{ name }}
+  git_rev: v{{ version }}
+
+build:
+  number: 0
+  script: "{{ PYTHON }} tools/ci_build/build.py --build_dir build/Linux --config Release --use_openmp --use_mkldnn --enable_pybind --build_wheel && pip install build/Linux/Release/dist/*.whl"
```

Comment:
As you have a ``build.sh`` you do not need this line, as conda will call that script